src/functions/instance/state/describeTableCache.js

```diff
@@ -0,0 +1,23 @@
+const descriptionCache = new Map(JSON.parse(sessionStorage.getItem('descriptionCache')) || []);
+
+const getTableKey = (schema, table) => `${schema}/${table}`;
+
+const getTableDescriptionFromCache = (schema, table) => {
+	const tableKey = getTableKey(schema, table);
+	if (!descriptionCache.has(tableKey)) return null;
+	return descriptionCache.get(tableKey);
+};
+
+const setTableDescriptionInCache = (schema, table, description) => {
+	const tableKey = getTableKey(schema, table);
+	descriptionCache.set(tableKey, description);
+	sessionStorage.setItem('descriptionCache', JSON.stringify(Array.from(descriptionCache)));
```

Comment:
I really don't think we even need or want to use session storage at all. The problem is that with this there is no timer for expiring the content, and it could be unbounded in how old the cached data is, until the 60 second timer fires a minute after the page loads. But on initial page load, we could end up showing very old cached table structure.

Reply:
Gotcha! I'll remove the use of sessionStorage 🙂